pkg/platform/kube/ingress/ingress.go

```diff
@@ -333,10 +333,12 @@ func (m *Manager) compileAnnotations(ctx context.Context, spec Spec) (map[string
 			enableSSLRedirect = *spec.EnableSSLRedirect
 		}
 
+		// if SSL redirect is enabled, set the annotation to true, otherwise set it to false unless it's already set
+		SSLRedirectAnnotation := "nginx.ingress.kubernetes.io/ssl-redirect"
```

Comment:
Maybe we can make it a const?

Reply:
I first made it a const, but then I felt like all nginx annotations should be consts, and then all other annotations consts....
Anyway it made this small PR bigger and unrelated, so decided to do it separately :)